tests/integration/targets/nxos_vrf_interfaces/tasks/main.yaml

```diff
@@ -1,12 +1,5 @@
 ---
-- name: Run the CLI and NX-API tests
-  block:
-    - name: Include the CLI tasks
-      ansible.builtin.include_tasks: cli.yaml
-      tags:
-        - cli
-  always:
-    - name: Include the NX-API tasks
-      ansible.builtin.include_tasks: nxapi.yaml
-      tags:
-        - nxapi
+- name: Main task for vrf_interfaces module
+  ansible.builtin.include_tasks: cli.yaml
+  tags:
+    - network_cli
```

Comment:
Why did we remove NX-API tests for this target?

Reply:
had changed it when local testing! reverted back